src/main/resources/application.conf

```diff
@@ -69,6 +70,9 @@ etc-client {
     chain-id = "3d"
 
     custom-genesis-file = null
+
+    // YP eq 150
+    block-reward = "5000000000000000000"
```

Comment:
@LukasGasior1 I was talking about changing this reward based on block number actually, do you think that makes sense or not? Maybe it's just a _feature_ that will never be used